spec/offchain.tex

```diff
@@ -115,21 +116,24 @@ \subsection{Variables}
 	\item $\hatmL$: UTxO set representing the local ledger state resulting from
 	      applying $\hatmT$ to $\bar{S}.U$ to validate requested transactions.
 	\item $\hatmT \in \mT^{*}$: List of transactions applied locally and pending
-	      inclusion in a snapshot (if this party is the next leader).
-	\item $\tx_{\omega} \in \mathcal{T}^{?}$: Pending decommit transaction, whose outputs are to be decommitted from the head. May be $\bot$ if nothing to decommit.
+		inclusion in a snapshot (if this party is the next leader).
+	\item \red{$\tx_\alpha \in \mathcal{T}$: Pending increment transaction, whose inputs are to be deposited to the head.}
+	\item $\tx_\omega \in \mathcal{T}$: Pending decrement transaction, whose outputs are to be withdrawn from the head.
 	\item $\bar{\mc S}$: Snapshot object of the latest confirmed snapshot which contains:
-	      \begin{center}
-		      \begin{tabular}{|l|l|}\hline
-			      $\bar{\mc S}.v$            & snapshot version                                              \\ \hline
-			      $\bar{\mc S}.s$            & snapshot number                                               \\ \hline
-			      $\bar{\mc S}.T$            & set of transactions relating this snapshot to its predecessor \\ \hline
-			      $\bar{\mc S}.U$            & snapshotted UTxO set                                          \\ \hline
-			      $\bar{\mc S}.\tx_{\omega}$ & optional decommit transaction                                 \\ \hline
-			      $\bar{\mc S}.\sigma$       & multisignature                                                \\ \hline
-		      \end{tabular}
-	      \end{center}
-	      The function $\Sno(v,n,T,U,\tx_{\omega})$ initializes a snapshot object with no multi-signature set.
-	      \todo{add types?}
+		\begin{center}
+			\begin{tabular}{|l|l|}\hline
+				$\bar{\mc S}.v$         & snapshot version \\ \hline
+				$\bar{\mc S}.s$         & snapshot number \\ \hline
+				$\bar{\mc S}.T$         & set of transactions relating this snapshot to its predecessor \\ \hline
+				$\bar{\mc S}.U$         & snapshotted UTxO set \\ \hline
+				\red{$\bar{\mc S}.U_\alpha = \mathsf{inputs}(\tx_\alpha)$}  & \red{pending UTxO to commit} \\ \hline
+				\red{$\bar{\mc S}.U_\omega = \mathsf{outputs}(\tx_\omega)$}  & \red{pending UTxO to decommit} \\ \hline
+				$\bar{\mc S}.\sigma$    & multisignature \\ \hline
+			\end{tabular}
+		\end{center}
+    The function $\Sno(v,n,U,T,\red{\mathsf{inputs}(tx_\alpha)}, \mathsf{outputs}(tx_\omega))$ initializes a snapshot object with no multi-signature set.
```

Comment:
Should: move the $\mathsf{inputs}(tx_\alpha)$ and $\mathsf{outputs}(tx_\omega)$ to the invocation